src/System.Private.Reflection.Metadata.Ecma335/src/Configurations.props

```diff
@@ -1,6 +0,0 @@
-﻿<Project>
-  <PropertyGroup>
-    <BuildConfigurations>
```

Comment:
So this project wasn't built at all?

Reply:
So can we remove that from the project exclusions now?

Reply:
I'm doing that in this PR

Reply:
Cool somehow missed it (how, right? With so little changes 😝)

Reply:
And it's super late for you 😛